### DISCUSSION TITLE
Error before make

### DISCUSSION THREAD

**Discussion by chadsantos** [Boos4721/openwrt#46] [Bugs/问题反馈]
Encountered this error,  is it normal? If not please advise.

root@Homedev:~# cd openwrt
root@Homedev:~/openwrt# make menuconfig
tmp/.config-package.in:113754:error: recursive dependency detected!
tmp/.config-package.in:113754:  symbol PACKAGE_wrtbwmon depends on BUSYBOX_DEFAULT_IP
package/utils/busybox/Config-defaults.in:2382:  symbol BUSYBOX_DEFAULT_IP depends on PACKAGE_busybox
tmp/.config-package.in:143:     symbol PACKAGE_busybox is selected by PACKAGE_wrtbwmon
For a resolution refer to Documentation/kbuild/kconfig-language.rst
subsection "Kconfig recursive dependency limitations"

tmp/.config-package.in:93592:error: recursive dependency detected!
tmp/.config-package.in:93592:   symbol PACKAGE_iptables is selected by PACKAGE_ip-full
tmp/.config-package.in:100260:  symbol PACKAGE_ip-full is selected by PACKAGE_luci-app-passwall
tmp/.config-package.in:66832:   symbol PACKAGE_luci-app-passwall depends on PACKAGE_iptables
For a resolution refer to Documentation/kbuild/kconfig-language.rst
subsection "Kconfig recursive dependency limitations"

configuration written to .config

*** End of the configuration.
*** Execute 'make' to start the build or try 'make help'.

**Comment by Boos4721** [comment]
it`s normal
